recirq/qml_lfe/circuit_blocks.py

```diff
@@ -0,0 +1,162 @@
+# Copyright 2020 Google
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Sycamore native circuit building blocks."""
+
+import os
+import cirq
+import cirq_google
+import functools
+
+
+@functools.lru_cache(maxsize=128)
+def _load_circuit(fname):
+    with open(os.path.join(os.path.dirname(__file__), fname), "r") as f:
+        return cirq.read_json(f)
+
+
+def tsym_block(qubits, params):
+    """Create a tsym block.
```

Comment:
What's a tsym block?  Will people know what this is?  (I don't).

Reply:
People will know what this is. We explain it a little more in the paper. https://en.wikipedia.org/wiki/T-symmetry